jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/PortForwardService.java

```diff
@@ -154,9 +155,15 @@ public void eventReceived(Action action, Pod pod) {
             }
 
             @Override
-            public void onClose(KubernetesClientException e) {
+            public void onClose() {
```

Comment:
Same as https://github.com/eclipse/jkube/pull/528#pullrequestreview-557863925

Please check if there are more of these.